django/core/handlers/asgi.py

```diff
@@ -268,7 +268,7 @@ async def send_response(self, response, send):
             #   allow mapping of a sync iterator.
             # - Use aclosing() when consuming aiter.
             #   See https://github.com/python/cpython/commit/6e8dcda
-            async with aclosing(response.__aiter__()) as content:
+            async with aclosing(aiter(response)) as content:
```

Comment:
Should we update the comment above? :thinking: 
```
Consume via aiter() ...
```

Reply:
Perhaps. I wasn't sure whether it was explicitly referring to the `.__aiter__()` of the response object when compared to `.streaming_content` on the same object. Feel free to choose and edit - I honestly don't know which to go for.

Reply:
I think the comment should be left (it's as @ngnpope says)